<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectionOrderDialog</class>
 <widget class="QDialog" name="InspectionOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Źleć inspekcje</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="3">
    <widget class="QDateEdit" name="dateEdit_PlannedDate"/>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QScrollArea" name="scrollArea_inspectors">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>433</width>
        <height>59</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btn_removeDriver">
     <property name="text">
      <string>usuń</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_searchDriver">
     <property name="text">
      <string>wyszukaj kierowce</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditDriver"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_searchInspector">
     <property name="text">
      <string>wyszukaj inspektora</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InspectionOrderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InspectionOrderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
